deploy/template.yaml

```diff
@@ -245,10 +279,10 @@ Resources:
               Resource:
                 - !ImportValue AuditEventQueueEncryptionKeyArn
       Events:
-        IPVCriUKPassportAPI:
+        IPVCriUKInternalPassportAPI:
```

Comment:
Just realised, as this stands this change will break the environment without simultaneously updating passport front config to point to the new api gateway. In order to split this API gateway without breaking the environment, rather than replacing this event with a reference to the new API gateway, let's add an extra event so this lambda can be invoked via the existing and the new API gateway. Deploy that change and then in each environment switch passport front config to point to the new "internal" api gateway. Once all switched over, remove the original event that references the `IPVCriUKPassportAPI`. eg.

```
Events:
        IPVCriUKInternalPassportAPI:
          Type: Api
          Properties:
            RestApiId: !Ref IPVCriUKInternalPassportAPI
            Path: /authorization
            Method: POST
        IPVCriUKPassportAPI:
          Type: Api
          Properties:
            RestApiId: !Ref IPVCriUKInternalPassportAPI
            Path: /authorization
            Method: POST
```

It would be good for us to get into the mindset of making these kinds of refactoring without impacting our environments.

Reply:
this should be done now as well.